handsontable/src/plugins/comments/comments.js

```diff
@@ -81,7 +81,6 @@ const META_READONLY = 'readOnly';
  * commentsPlugin.removeComment();
  * ```
  */
-/* eslint-enable jsdoc/require-description-complete-sentence */
```

Comment:
Same as above, removing this line makes the entire file being ignored by the eslint, which is probably not the best approach.

Reply:
The complete-sentence rule is not useful and causes legibility problems. Can we just turn it off for all JSDocs?